### PR TITLE
Fix for incorrect order of classNames fixes #1058

### LINKED ISSUE #1058: Kit Classname Order Affects Kit Styles
**Describe the bug**

The Icon Value kit (potentially others) is placing the kit class name last instead of first. This hides other CSS styles that have not been working until now. 

**To Reproduce**

1. Use the kit with a custom class name

**Expected behavior**
Passing `className` should not change the behavior of the kit's style cascading

**Screenshots**

Screen Shot 2020-09-02 at 1 48 56 PM
Screen Shot 2020-09-02 at 1 47 41 PM

**Additional context**
none

### PR DESCRIPTION
**Pull request by jasoncypret**
#### Screens

Non-visual change

#### Breaking Changes

Nope

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-142

#### How to test this

This is mainly a dev fix to ensure custom class names that are added to a kit display correctly. Confirm the kits load as expected. The bug was reported in #1058 on the installer schedule, so perhaps you can see the extra padding go away.

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `depreciation`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.

### CODE REVIEW

**Review by thestephenmarshall — APPROVED** [review]

**Comment by thestephenmarshall** [comment]
@jasoncypret Any alpha testing needed for this one?

**Comment by jasoncypret** [comment]
@thestephenmarshall We can but it will be hard to test. I wouldn't know where to look. We would need to find all the places in Nitro that have added a classname to a kit, even better if it's using CSS modules.

A quick test could be check install schedules...but I'm not sure the problem exists anymore there now. This fixes #1058 

@stephenagreer Care to weigh in here?

File: app/pb_kits/playbook/pb_form_pill/_form_pill.jsx [hunk 1]
```diff
@@ -18,8 +18,9 @@ type FormPillProps = {
 const FormPill = (props: FormPillProps) => {
   const { className, text, name, onClick = () => {}, avatarUrl } = props
   const css = classnames(
-    [`pb_form_pill_kit_${'primary'}`, className],
-    globalProps(props)
+    [`pb_form_pill_kit_${'primary'}`],
```
Comment (drborges) [inline app/pb_kits/playbook/pb_form_pill/_form_pill.jsx:21]:
```suggestion
    `pb_form_pill_kit_${'primary'}`,
```

File: app/pb_kits/playbook/pb_currency/_currency.jsx [hunk 2]
```diff
@@ -45,7 +45,11 @@ const Currency = (props: CurrencyProps) => {
   const [whole, decimal = '00'] = amount.split('.')
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
-  const classes = classnames(buildCss('pb_currency_kit', align, size, { dark: dark }), className, globalProps(props))
+  const classes = classnames(
+    buildCss('pb_currency_kit', align, size, { dark: dark }),
```
Comment (drborges) [inline app/pb_kits/playbook/pb_currency/_currency.jsx:49]:
```suggestion
    buildCss('pb_currency_kit', align, size, { dark }),
```

File: app/pb_kits/playbook/pb_checkbox/_checkbox.jsx [hunk 3]
```diff
@@ -40,8 +40,11 @@ const Checkbox = (props: CheckboxProps) => {
 
   const dataProps = buildDataProps(data)
   const ariaProps = buildAriaProps(aria)
-  const classes = classnames(buildCss('pb_checkbox_kit',
-    { checked: checked, error: error }), className, globalProps(props))
+  const classes = classnames(
+    buildCss('pb_checkbox_kit', { checked: checked, error: error }),
```
Comment (drborges) [inline app/pb_kits/playbook/pb_checkbox/_checkbox.jsx:44]:
```suggestion
    buildCss('pb_checkbox_kit', { checked, error }),
```

File: app/pb_kits/playbook/pb_badge/_badge.jsx [hunk 4]
```diff
@@ -32,8 +32,10 @@ const Badge = (props: BadgeProps) => {
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
   const css = classnames(
-    buildCss('pb_badge_kit', variant, { rounded: rounded }), className,
-    globalProps(props))
+    buildCss('pb_badge_kit', variant, { rounded: rounded }),
```
Comment (drborges) [inline app/pb_kits/playbook/pb_badge/_badge.jsx:35]:
```suggestion
    buildCss('pb_badge_kit', variant, { rounded }),
```

File: app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.jsx [hunk 5]
```diff
@@ -35,7 +35,11 @@ const ButtonToolbar  = (props: ButtonToolbarProps) => {
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
 
-  const classes = classnames(buildCss('pb_button_toolbar_kit', orientation, { connected: connected }), className, globalProps(props))
+  const classes = classnames(
+    buildCss('pb_button_toolbar_kit', orientation, { connected: connected }),
```
Comment (drborges) [inline app/pb_kits/playbook/pb_button_toolbar/_button_toolbar.jsx:39]:
```suggestion
    buildCss('pb_button_toolbar_kit', orientation, { connected }),
```

File: app/pb_kits/playbook/pb_time_stacked/_time_stacked.jsx [hunk 6]
```diff
@@ -26,11 +26,11 @@ type TimeStackedProps = {
 const TimeStacked = (props: TimeStackedProps) => {
   const { className, dark = false, date, tag = 'body' } = props
   const classes = classnames(
-    className,
     buildCss('pb_time_stacked_kit', {
       dark: dark,
```
Comment (drborges) [inline app/pb_kits/playbook/pb_time_stacked/_time_stacked.jsx:30]:
```suggestion
      dark,
```

**Review by drborges — COMMENTED** [review]
I've got small suggestions, but nothing that must be addressed.

**Pull request merged**